boards/arm/nucleo_h745zi_q/Kconfig.defconfig

```diff
@@ -43,4 +43,24 @@ choice STM32H7_DUAL_CORE_BOOT
 	depends on STM32H7_DUAL_CORE
 endchoice
 
+if NETWORKING
+
+config NET_L2_ETHERNET
+	default y
+
+config ETH_STM32_HAL
+	default y
+
+config NOCACHE_MEMORY
+	# nocache memory needed for
+	# proper ETH_DMA transitions
+	default y
+
+config USE_STM32_HAL_RCC_EX
```

Comment:
Does board code need to access RCC? Sounds like this is not in the right place

Reply:
Yes, it needed to access RCC_EX (see [RCC_EX support issue](https://github.com/zephyrproject-rtos/hal_stm32/pull/56) in **hal_stm32** thread).
@gmarull What would be your proposal?

Reply:
@Nukersson, should be done in drivers/ethernet/Kconfig.stm32_hal:
```
menuconfig ETH_STM32_HAL
	bool "STM32 HAL Ethernet driver"
	select USE_STM32_HAL_ETH
+     select USE_STM32_HAL_RCC_EX if ....
	help
           ...
```

Reply:
@erwango Thank you for very helpful advice. The corresponding definition were moved to suggested KConfig file.